aiida_vasp/parsers/settings.py

```diff
@@ -23,86 +23,102 @@
         'DOSCAR': {
             'parser_class': DoscarParser,
             'is_critical': False,
-            'status': 'Unknown'
+            'status': 'Unknown',
+            'open_mode': 'r'
```

Comment:
Thanks. Yes, maybe. Since a parser only works on one file this should be fine. As it is now would allow a parser to e.g. parse two different files with different mode. But we do not use that and will not use it I think. I am moving it to the parser for now as we we should not open up for functionality that we do not really see a need for. And yes, maintenance would be more simple.